dask_kubernetes/tests/test_sync.py

```diff
@@ -318,7 +316,7 @@ def test_scale_up_down(cluster, client):
 
     a, b = list(cluster.scheduler_info["workers"])
     x = client.submit(np.ones, 1, workers=a)
-    y = client.submit(np.ones, 50_000, workers=b)
+    y = client.submit(np.ones, 50000, workers=b)
```

Comment:
This was set intentionally using the underscore notation for readability. Is there a reason why you changed it?

Reply:
Oops, may have to do with my black settings - I will check!